codes/classical/bits/repetition code.yml

```diff
@@ -0,0 +1,90 @@
+
+# This field starts a list of specific labeled subfields; do not leave it
+# empty. If empty, comment out. Also, indentations are important!
+features:
+
+  # The fields below relate to specific properties of the code that are worth
+  # noting. In some cases (see below), there might be multiple pieces of
+  # information associated with a field (e.g., mutliple encoders/decoders,
+  # multiple facts about fault tolerance, etc.); in such cases, the field
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: 'The code rate is \(\frac{1}{n}\). The code distance is \(n\). Its generator matrix is \(G=\left[\begin{smallmatrix}1 & 1&\cdots& 1 & 1 \end{smallmatrix}\right]\). '
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  encoders: 
+    - 'The encoding process is \(1\rightarrow 111...1\), \(0\rightarrow 000...0\). Repeating the information \(n\)-times.'
+
+  decoders:
+    - 'Calculate the Hamming weight \(d_H\) of the code. If \(d_H\leq \frac{n-1}{2}\), decode the code as 0. If \(d_H\geq \frac{n+1}{2}\), decode the code as 1.'
+
+
+  threshold: 
+    - 'Suppose each bit has probability \(p\) of being received correctly, independent for each bit. The probability that a repetition code is received correctly is \(\sum_{k=0}^{(n-1)/2}\frac{n!}{k!(n-k)!}p^{n-k}(1-p)^{k}\).'
+
+  
+  # Propose a field that you feel is applicable to a large and interesting class of codes
+  #some_other_field:
+  # - 'detail'
+
+realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  - 'Although classical repetition code does not have many real-world applications, it still inspire the studies of quantum error correction. Quantum repetition code is widely studied in quantum computing community. For example, the 3-qubit repetition code is the simplest quantum error correcting code. Moreover, the 9-qubit Shor code is concatenation of 3-qubit bit-flip code and 3-qubit phase-flip code which corrects arbitrary single qubit errors.'
+
+notes:
+  - '\(n\) must be an odd number.'
+  - 'Can be used to concatenate other codes.'
+  - 'The code rate is low.'
+  - 'Widely used in quantum error correction.'
+
+#relations:
+  #parents:
+    #- code_id: Hamming
+      #detail: 'A \([n,1,n]\) repetition code is a (n,1)-Hamming code.'
+
+  #cousins:
+    #- code_id: Shor_code
+      #detail: 'Shor 9-qubit code is a concatenation code of 3-qubit bit-flip code and 3-qubit phase-flip code.'
+    #- code_id: Concatented_repetition_binomial
```

Comment:
Good observation. I've moved this to quantum_repetition; please remove.

Reply:
Thanks for the comments! The issues of repetition code are addressed.